pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -50,13 +62,530 @@ func New(_ runtime.Object, fh framework.FrameworkHandle) (framework.Plugin, erro
 
 // PostFilter invoked at the postFilter extension point.
 func (pl *DefaultPreemption) PostFilter(ctx context.Context, state *framework.CycleState, pod *v1.Pod, m framework.NodeToStatusMap) (*framework.PostFilterResult, *framework.Status) {
-	nnn, err := core.Preempt(ctx, pl.fh, state, pod, m)
+	nnn, err := Preempt(ctx, pl.fh, state, pod, m)
 	if err != nil {
 		return nil, framework.NewStatus(framework.Error, err.Error())
 	}
 	if nnn == "" {
 		return nil, framework.NewStatus(framework.Unschedulable)
-	} else {
-		return &framework.PostFilterResult{NominatedNodeName: nnn}, framework.NewStatus(framework.Success)
 	}
+	return &framework.PostFilterResult{NominatedNodeName: nnn}, framework.NewStatus(framework.Success)
+}
+
+// Preempt finds nodes with pods that can be preempted to make room for "pod" to
```

Comment:
I am guessing all this logic was simply moved, nothing at all changed, right?

Reply:
Yes, just code movement. A dependent PR (#92107) also just moves testing stuff to pkg/scheduler/testing.